api/docs/http_catalog.md

```diff
@@ -8,7 +8,7 @@ Find
 
 Retrieve a catalog.
 
-    POST /puppet/v3/catalog/:nodename?environment=:environment
+    POST /puppet/v3/catalog/:nodename
```

Comment:
Yeah, changed it because you can't provide query strings on a POST. I'll update the Parameters section.